src/scancode/cli.py

```diff
@@ -536,7 +536,7 @@ def scan(input_path,
          scans_cache_class=None,
          strip_root=False,
          full_root=False,
-         pre_scan_plugins=()):
+         pre_scan_plugins=None):
```

Comment:
Giving an empty tuple as a default parameter gives the reader an idea of what type of data to expect. Why did you change it?

Reply:
Well, it gives a wrong idea to me since the `pre_scan_plugins` is actually a list.
I should have explain it in the commit message, sorry.

Reply:
It's not a good idea to pass mutable objects as default parameters. Using a tuple gives an idea that a collection is passed here. :)
BTW I didn't know this myself till @pombredanne posted a similar comment in an earlier review.
I'll try to find that for you. :P

Reply:
I know that. That's why I changed it to `None` and not `[]`.

Reply:
I might prefer using an empty tuple as an hint that the expected value is an iterable ofsorts, but that's a weak indication for sure... @jpopelka your way works too.

Reply:
@jpopelka to your point, confusing is not a good thing, hence why I went ahead and merged it as-is :) Thanks again!